Java/ds/Intersection_of_two_linkedlists.java

```diff
@@ -0,0 +1,140 @@
+import java.util.*;
+class Intersection_of_two_linkedlists {
+    static class Node 
+    {
+        int data;
+        Node next; 
+        Node(int x)
+        {
+            data = x;
+            next = null;
+        }
+    }
+ 
+    static Node head1;
+    static Node head2;
+     
+    public void append1(int newdata) 
+    { 
+        Node newnode = new Node(newdata); 
+        if (head1 == null) 
+        { 
+            head1 = new Node(newdata); 
+            return; 
+        } 
+  
+        newnode.next = null; 
+ 
+        Node last = head1;  
+        while (last.next != null) 
+            last = last.next; 
+  
+        last.next = newnode; 
+        return; 
+    }
+ 
+    public void append2(int data) 
+    { 
+        Node newnode = new Node(data); 
+        if (head2 == null) 
+        { 
+            head2 = new Node(data); 
+            return; 
+        } 
+  
+        newnode.next = null; 
+ 
+        Node last = head2;  
+        while (last.next != null) 
+            last = last.next; 
+  
+        last.next = newnode; 
+        return; 
+    }
+    
+    
+    int GetNode() //function to get the intersection point of two linked lists
```

Comment:
Use Comments above the Line of Code. Not beside it.

Reply:
Okay sir, it's updated now.